src/maud/utils.py

```diff
@@ -68,51 +69,125 @@ def get_rref(mat):
     return sp.Matrix(mat).rref(iszerofunc=lambda x: abs(x) < 1e-10)[0]
 
 
-def export_scaled_params_from_draws(infd, chain, draw):
-    """Extact log centered parameters from an infd object."""
-    list_of_input_inits = [
-        "log_km_z",
-        "drain_z",
-        "log_ki_z",
-        "log_dissociation_constant_t_z",
-        "log_dissociation_constant_r_z",
-        "log_transfer_constant_z",
-        "log_kcat_z",
-        "log_phos_kcat_z",
-        "log_conc_unbalanced_z",
-        "log_enzyme_z",
-        "log_phos_conc_z",
-        "formation_energy_z",
-    ]
-
-    input_dict = {
-        par_name: infd.posterior[par_name][chain, draw].values.tolist()
-        for par_name in list_of_input_inits
-        if par_name in infd.posterior.variables.keys()
-    }
-    return input_dict
-
-
 def export_params_from_draw(infd, chain, draw):
     """Extact parameters from an infd object."""
     list_of_input_inits = [
         "km",
         "drain",
         "ki",
-        "dissociation_constant_t",
-        "dissociation_constant_r",
+        "diss_t",
+        "diss_r",
         "transfer_constant",
         "kcat",
-        "phos_enzyme_kcat",
+        "kcat_phos",
         "conc_unbalanced",
-        "enzyme",
-        "phos_enzyme_conc",
-        "formation_energy",
+        "conc_enzyme",
+        "conc_phos",
+        "dgf",
     ]
 
     input_dict = {
         par_name: infd.posterior[par_name][chain, draw].values.tolist()
         for par_name in list_of_input_inits
         if par_name in infd.posterior.variables.keys()
     }
+
     return input_dict
+
+
+def get_input_template(km, raw_measurements):
```

Comment:
I'm not sure about the extra import, class definition and nested function definition living inside this function, as well as the hardcoded column list that you need to read to the end in order to discover. I think all these issues could be solved while avoiding jumbling up the `utils.py` file by putting this logic in it's own file (say `get_input_template.py`).